File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// ColoradoInformationMarketplace SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class ColoradoInformationMarketplaceMakeContext
{
    public static function call(array $ctxmap, ?ColoradoInformationMarketplaceContext $basectx): ColoradoInformationMarketplaceContext
    {
        return new ColoradoInformationMarketplaceContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# ColoradoInformationMarketplace SDK utility: make_context
require_relative '../core/context'
module ColoradoInformationMarketplaceUtilities
  MakeContext = ->(ctxmap, basectx) {
    ColoradoInformationMarketplaceContext.new(ctxmap, basectx)
  }
end
